playbooks/10 - SP - GDPR Framework/Get Updated Data Breach Report from DPO.json

```diff
@@ -581,14 +581,14 @@
                     "mitigationActions": "{{vars.steps.Update_DPO_Report.input['whatMitigationActionsHaveBeenImplemented?']}}",
                     "noOfAffectedUsers": "{{vars.steps.Update_DPO_Report.input['no.OfAffectedUsers']}}",
                     "whereDataWasHosted": "{{vars.steps.Update_DPO_Report.input['whereDataWasHosted?']}}",
-                    "wasTheDataEncrypted": "{{\"ChooseYesOrNo\" | picklist(vars.steps.Update_DPO_Report.input['wasTheDataEncrypted?'], \"@id\")}}",
+                    "wasTheDataEncrypted": "{{\"YesNoInput\" | picklist(vars.steps.Update_DPO_Report.input['wasTheDataEncrypted?'], \"@id\")}}",
```

Comment:
How about "isDataEncrypted"

Reply:
This is a standard GDPR Data Breach Notification Template, all questions are in the past tense. And refers to the time when a data breach has happened instead of the current state